integration-tests/Makefile

```diff
@@ -22,43 +22,43 @@ docker-clean:
 tests: docker-clean
 	go version
 	set -o pipefail ; \
-	go test -timeout 30m -count=1 -v \
+	go test -timeout 60m -count=1 -v \
```

Comment:
We might want to bump the `no_output_timeout` value in the CI configuration to something like 75 minutes. Though I have to say that a full hour to run some tests seems a little excessive for regular testing, any way we could have this configurable? Also, if this is not failing in CI then we might not need to bump the timeout at all 🤷🏻‍♂️ 
https://github.com/stackrox/collector/blob/452355b4d3c90247d176f16c392d7529b72a753b/.circleci/config.yml#L983

Reply:
This is purely a fail safe, and one that we inconsistently hit on CI when running the (fairly heavy weight) syscall latency measurements, so it feels reasonable to bump it. I think the no_output_timeout should also remain the same since the tests should be outputting some info pretty frequently and no output would suggest some breakage somewhere.